src/app/technical-record/body/body.component.html

```diff
@@ -1,5 +1,9 @@
 <dl class="technical-record-sub-details govuk-summary-list ">
```

Comment:
please we need to remove this css class if not used `technical-record-sub-details`